docs/readmes/lte/architecture_overview.md

```diff
@@ -29,7 +29,50 @@ Note: The *mme* service will be renamed soon to *accessd*, as its main purpose i
 
 Together, these components help to facilitate and manage data both to and from the user.
 
-![AGW diagram](assets/agw_services.png)
+
+```
+     ┏━━━━━━━━━━━━━━┓           ┏━━━━━━━━━━━━━━┓
+     ┃ Magma Orc8r  ┃───────────┃  Magma Feg   ┃
+     ┗━━━━━━━━━━━━━━┛           ┗━━━━━━━━━━━━━━┛
+             ▲      ▲
+             │      │
+       ┏━━━━━┻━━━━━━┻━━━━━┓
+╔══════╣  control_proxy   ╠═══════════════════════════════════════════Access Gateway (AGW)
+║      ┗━━━━━┳━━━━━━┳━━━━━┛                                                              ║
+║            │      └────────────────────────────────────────────────┐                   ║
+║            └──────────────────────────────────┐                    │                   ║
+║                                               │                    │                   ║
+║ ┏━━━━Magma EPC━━━━┓         ┏━━━━Session━━━━┓ │  ┏━━━━Device━━━━━┓ │┏━━━Optional━━━━┓  ║
+║ ┃   ┌───────────┐ ┃         ┃   & Policy    ┃ │  ┃  Management   ┃ │┃   Services    ┃  ║
+║ ┃┌─▶│    MME    │◀╋───────┐ ┃  Management   ┃ │  ┃ ┌───────────┐ ┃ │┃ ┌────────────┐┃  ║
+║ ┃│  └───────────┘ ┃       │ ┃ ┌───────────┐ ┃ └──╋─│  magmad   │ ┃ │┃ │  monitord  │┃  ║
+║ ┃│  │     │     ──╋────┐  │ ┃ │ policydb  │◀╋┐   ┃ └───────────┘ ┃ │┃ └────────────┘┃  ║
+║ ┃│  │     │       ┃    │  │ ┃ └───────────┘ ┃│   ┃ ┌───────────┐ ┃ │┃ ┌───────────┐ ┃  ║
```

Comment:
enodebd is also part of device management.

Reply:
I would not categorize directoryd and mobilityd as Subscriber Management. Don't have a good alternative name to consider.

Reply:
> I would not categorize directoryd and mobilityd as Subscriber Management. Don't have a good alternative name to consider.

Should I just remove the subscriber Management grouping box and leave all three components `directoryd`, `mobilityd` and `subscriberdb` free-floating?

Reply:
I would prefer them floating.

Reply:
I'll float them! Thanks for the feedback @ulaskozat - I think I had auto-merge on so I'll fix this now in a follow up PR.

Reply:
https://github.com/magma/magma/pull/6123